src/components/ZeplinPanel.tsx

```diff
@@ -22,6 +23,7 @@ interface ZeplinState {
     zoomLevel: number;
     loading: boolean;
     error: string | null;
+    connectedComponents: string[] | null;
```

Comment:
These seems to filled up only with Zeplin links, could you rename the related variables and function names to indicate it?